### ISSUE TITLE
How to finetune darknet19_448 classifier

### ISSUE THREAD

**Issue by lixiangchun** [pjreddie/darknet#398]
I want to finetune darknet19_448 on my data set with 2 classes.

I changed the the number of filters of the last convolutional layers in ```darknet19_488.cfg``` to ```filters = 2```. 

In the ```my.data``` file, the number of classes and path to training and validation data sets were set accordingly.
```
classes = 2
train = train.list
val = val.list
labels = mylabels.txt
backup = backup_darknet19_448
```

I run the command line as:
```
darknet classifier train cfg/my.data cfg/darknet19_448.cfg darknet19_448.weights
```
Is that correct?

**Comment by lqian** [comment]
First you need to change the last conv layer filters number to 
`filters=2`
in line 187 of file https://github.com/pjreddie/darknet/blob/master/cfg/darknet19_448.cfg.
Then you should train the network without any pre-trained weights.

**Comment by encoras** [comment]
Dear lqian,
Can you explain oh how to create a "labels = mylabels.txt" for my 10 class data in format:
D:/Broko_Dataset/Class0/Train/1900.jpg 
D:/Broko_Dataset/Class0/Train/2300.jpg 
D:/Broko_Dataset/Class4/Train/1100.jpg 
D:/Broko_Dataset/Class5/Train/1100.jpg 
D:/Broko_Dataset/Class6/Train/1000.jpg 
D:/Broko_Dataset/Class7/Train/1300.jpg

During the training I get notification "too much or too few labels".
How to write a correct labels.txt? My class names are 0,1,2,3,...,9
Regards,

**Comment by andeyeluguo** [comment]
I have the same problem.
Couldn't open file: data/cifar/labels.txt
but I can open it by vim, and its content is like this
airplane
automobile
bird
cat
deer
dog
frog
horse
ship
truck

I don't know why.

**Comment by SouradipBh** [comment]
How we finetune the darknet model for 12 classes?
Tell me what we need to change in the cfg file(for custom object),which parameters are related with finetune?
